refactor-first-maven-plugin/src/main/java/org/hjug/mavenreport/RefactorFirstJsonReport/JsonReportDisharmonyEntry.java

```diff
@@ -0,0 +1,40 @@
+package org.hjug.mavenreport.RefactorFirstJsonReport;
+
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.Locale;
+
+import lombok.Builder;
+import lombok.Data;
+import org.hjug.cbc.RankedDisharmony;
+
+@Data
+@Builder
+class JsonReportDisharmonyEntry
+{
```

Comment:
I've always been partial to having opening curly braces on the same line - could you move the curly brace up?
e.g. 
``` Java
class JsonReportDisharmonyEntry {
```

Reply:
done